### DISCUSSION TITLE
Notification and icon dependencies

### DISCUSSION THREAD

**Discussion by t1r** [KasperskyLab/uif#68] [Q&A]
App with deps:
`"@kaspersky/components": "6.42.1",`
and using 
`<Notification .../>`
without deps:
`"@kaspersky/icons": "2.3.0"`
 doesn't work, can't find icons, is it ok?

**Comment by vostrik** [comment]
Ou, looks like bug. Thanks for your attention!

Could you add issue about this? We will fix it

**Reply by t1r** [reply]
Done 
https://github.com/KasperskyLab/uif/issues/69
